actors/miner/src/lib.rs

```diff
@@ -4302,48 +4207,140 @@ fn verify_windowed_post(
     Ok(result.is_ok())
 }
 
-fn get_verify_info(
-    rt: &impl Runtime,
-    params: SealVerifyParams,
-    unsealed_cid: CompactCommD,
-) -> Result<SealVerifyInfo, ActorError> {
-    if rt.curr_epoch() <= params.interactive_epoch {
-        return Err(actor_error!(forbidden, "too early to prove sector"));
+struct SealVerifyInfoInput {
```

Comment:
SectorProofInputs? SectorCommitmentInputs? SectorCommitmentProofInputs?

(Compare with proposed ReplicaUpdateInputs in #1376. Unfortunately the `ReplicaUpdateInfo` type, which is the proof inputs, isn't defined here so I couldn't rename)